docs/examples/notebooks/webdataset_reader_example.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ccbebed3-9329-4e75-ba38-b06574891cf5",
+   "metadata": {},
+   "source": [
+    "# Webdataset reader example in rocAL\n",
+    "This example demonstrates how to set up a simple webdataset reader pipeline that loads and decodes image data stored in webdataset format using rocAL."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1227f440-4c24-4e20-9c96-9a167856b081",
+   "metadata": {},
+   "source": [
+    "<font size=\"12\"> Common Code </font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7299da1c-e1f0-4672-8253-086b8a997b7e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from amd.rocal.plugin.generic import ROCALClassificationIterator\n",
+    "from amd.rocal.pipeline import pipeline_def\n",
+    "import amd.rocal.fn as fn\n",
+    "import amd.rocal.types as types\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7267c8c0",
+   "metadata": {},
+   "source": [
+    "## Using fn.readers.webdataset operator\n",
+    "\n",
+    "Data stored in WebDataset format is read using `fn.readers.webdataset`. `fn.readers.webdataset` takes the following arguments:\n",
```

Comment:
this looks confusing on the final notebook. Please say "is read using `fn.readers.webdataset` function. It takes the following arguments:"